src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs

```diff
@@ -206,6 +206,10 @@ internal static int GetWarningLevel(ErrorCode code)
             // docs/compilers/CSharp/Warnversion Warning Waves.md
             switch (code)
             {
+                case ErrorCode.WRN_AddressOfInAsync:
```

Comment:
Was the warning wave document updated correspondingly? 
https://github.com/dotnet/roslyn/blob/main/docs/compilers/CSharp/Warnversion%20Warning%20Waves.md

Reply:
Opened #67420